app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -86,4 +86,30 @@ class PatientEntryUpdateTest {
         )
   }
 
+  @Test
+  fun `when the user enters phone number which is too short, then show error`() {
+    //given
```

Comment:
Not necessary to change this, but we can skip the `// given`, `// when`, and `// then` sections for mobius `Init` and `Update` tests since they are a part of the `Spec` test helpers.

Reply:
This applies to all the tests in this PR.

Reply:
Okay resolved

Reply:
resolved